src/pocketmine/command/CommandReader.php

```diff
@@ -91,6 +95,10 @@ public function run(){
 
 			$lastLine = microtime(true);
 		}
+		global $stdin;
+		if(is_resource($stdin)){
+			fclose($stdin);
+		}
```

Comment:
@KAGsundaram you could use GitHub comment reactions. They can prevent this from occupying UI space or sending unnecessary notifications to other people.